test/fixtures/handlers.yml

```diff
@@ -2,7 +2,7 @@ email_deliver:
   rule: email
   service_name: 'Handlers::Emailer'
   settings:
-    recipients: "test+to@example.com"
-    from: "test+from@example.com"
-    subject: "New customer registered"
-    body: "New customer registered from Canada with id {{ id }} and email {{ email }}."
+    :recipients: "test+to@example.com"
+    :from: "test+from@example.com"
+    :subject: "New customer registered"
+    :body: "New customer registered from Canada with id {{ id }} and email {{ email }}."
```

Comment:
We persist them as symbols, so this is just for the fixture to mimic what's happening in real life. I think?